docs/postman/Smart Village API.postman_collection.json

```diff
@@ -307,6 +308,27 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "Membership - List",
+					"request": {
+						"method": "GET",
+						"header": [],
```

Comment:
is the bearer token missing? only registered users should be able to make the request. it seems to be missing here like for the other endpoint mentioned in #46.

Reply:
Hi @donni106 
You can pass the bearer token in request and save it

Reply:
But does the endpoint return an error without bearer token? For example if guests without login would try to fetch.

Reply:
I have tested it and it works well without bearer token:

```
{
  "name": "Unauthorized",
  "message": "Invalid token!",
  "code": 0,
  "status": "401",
  "type": "yii\\web\\HttpException"
}
```